js/uiaction.js

```diff
@@ -257,12 +259,29 @@ $(".deleteBtn").click(function(){
     hideWidgets();
 });
 
+// Delete using delete key or backspace key from keyboard- modified above code using keydown / keycode
+$('html').keydown(function(e) {
+    if ( isLabelBox( $(".selected") ) ){
+        deleteLabelBox($(".selected"));
+    }else if( isFeaturePoint ( $(".selected") ) ){
+        deleteFeaturePoint($(".selected"));
+    }else{
+        throw Error("Select a Label box or feature point to delete.");
```

Comment:
Please don't throw any error. Users of the application will not check the console and there may be a possibility that in future, multiple keys are associated for different purpose.